src/main/java/org/folio/converter/QuickMarcToParsedRecordConverter.java

```diff
@@ -58,56 +62,47 @@ private Record quickMarcJsonToMarcRecord(QuickMarcJson quickMarcJson){
         DataField dataField = factory.newDataField();
         dataField.setTag(field.getTag());
         dataField.getSubfields().addAll(stringToSubfields(field.getContent().toString()));
-        dataField.setIndicator1((field.getIndicators().get(0) == null)? ' ': field.getIndicators().get(0).toString().charAt(0));
-        dataField.setIndicator2((field.getIndicators().get(1) == null)? ' ': field.getIndicators().get(1).toString().charAt(0));
+        dataField.setIndicator1((field.getIndicators().get(0) == null)? SPACE_CHARACTER: field.getIndicators().get(0).toString().charAt(0));
+        dataField.setIndicator2((field.getIndicators().get(1) == null)? SPACE_CHARACTER: field.getIndicators().get(1).toString().charAt(0));
         marcRecord.getDataFields().add(dataField);
       }
     });
     return marcRecord;
   }
 
-  private String restoreField008(Map map) {
+  @SuppressWarnings("unchecked")
```

Comment:
This annotation shouldn't be used here! If you can't eliminate a warning and you can prove that the code that provoked the warning is typesafe, then (and only then) suppress the warning with an @SupressWarnings("unchecked") annotation. Please, consider another solution.